lstmcpipe/utils.py

```diff
@@ -80,7 +186,193 @@ def dump_lstchain_std_config(filename='lstchain_config.json', allsky=False, over
+            "train_test_splitting": getattr(self, "set_train_test_splitting_default_options"),
+            "train_pipe": getattr(self, "set_trainpipe_default_options"),
+            "RF_importance": getattr(self, "set_train_plot_rf_feat_default_options"),
+            "dl1_to_dl2": getattr(self, "set_dl1_dl2_default_options"),
+            "dl2_to_irfs": getattr(self, "set_dl2_irfs_default_options"),
+            "dl2_sens": getattr(self, "set_dl2_sens_default_options"),
+            "dl2_sens_plot": getattr(self, "set_dl2_sens_plot_default_options"),
+        }
+        _default_options[stage]()
+
+    def submit(self):
+        if self.wrap_cmd is None or self.wrap_cmd == "":
+            raise ValueError(
+                "You must first define the command to be batched: " "SbatchLstMCStage().wrap_command('COMMAND')"
+            )
+        else:
+            jobid = run_command(self.slurm_command)
+            return jobid
+
+    def r0_dl1_options(self, process_dl1_job_name="r0_dl1", array="0-0%100", partition="long", extra_slurm_options=""):
+        self.job_name = f"--job-name={process_dl1_job_name}"
+        self.slurm_options = f"--partition={partition} --array={array} {extra_slurm_options}"
+
+    def dl1ab_options(self, process_dl1ab_job_name="dl1ab", array="0-0%100", partition="long", extra_slurm_options=""):
+        self.job_name = f"--job-name={process_dl1ab_job_name}"
+        self.slurm_options = f"--partition={partition} --array={array} {extra_slurm_options}"
+
+    def set_merge_dl1_default_options(self):
+        self.job_name = "--job-name=merge"
+        self.slurm_partition = "--partition=long"
+
+    def set_train_test_splitting_default_options(self):
+        self.job_name = "--job-name=train_test_splitting"
+        self.slurm_partition = "--partition=short"
+
+    def set_trainpipe_default_options(self):
+        self.job_name = "--job-name=train_pipe"
+        # self.slurm_options = " --partition=long --mem=32G"
+        self.slurm_options = "--partition=xxl --mem=100G --cpus-per-task=16"
+
+    def set_train_plot_rf_feat_default_options(self):
+        self.job_name = "--job-name=RF_importance"
+        self.slurm_options = "--partition=short --mem=16G"
+
+    def set_dl1_dl2_default_options(self):
+        self.job_name = "--job-name=dl1_2"
+        self.slurm_options = "--partition=short --mem=32G"
+
+    def set_dl2_irfs_default_options(self):
+        self.job_name = "--job-name=dl2_IRFs"
+        self.slurm_partition = "--partition=short"
+
+    def set_dl2_sens_default_options(self):
+        self.job_name = "--job-name=dl2_sens"
+        self.slurm_options = "--partition=short --mem=32G"
+
+    def set_dl2_sens_plot_default_options(self):
+        self.job_name = "--job-name=dl2_sens_plot"
+        self.slurm_partition = "--partition=short"
+
+    # def load_from_log(self, jobid):
+    #     # TODO
+    #     jobid = ''
+    #     NotImplementedError("Sorry, we are working on it")
```

Comment:
to remove?

Reply:
If it was something to implement later, it was fine to leave it there, I wasn't sure if it was a left out